packages/teleterm/src/ui/StatusBar/ShareFeedback/useShareFeedback.ts

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { makeEmptyAttempt, useAsync } from 'shared/hooks/useAsync';
+import { ShareFeedbackFormValues } from './types';
+
+export const FEEDBACK_TOO_LONG_ERROR = 'FEEDBACK_TOO_LONG_ERROR';
+
+export function useShareFeedback() {
+  const ctx = useAppContext();
+  ctx.workspacesService.useState();
+  ctx.clustersService.useState();
+
+  const [isShareFeedbackOpened, setIsShareFeedbackOpened] = useState(false);
+  const [formValues, setFormValues] = useState<ShareFeedbackFormValues>({
+    feedback: '',
+    company: '',
+    email: getEmailFromUserName() || '',
+    newsletterEnabled: false,
+    salesContactEnabled: false,
+  });
+
+  const [submitFeedbackAttempt, submitFeedback, setSubmitFeedbackAttempt] =
+    useAsync(makeSubmitFeedbackRequest);
+
+  async function makeSubmitFeedbackRequest(): Promise<Response> {
+    preValidateForm();
+
+    const formData = new FormData();
+    formData.set('OS', ctx.mainProcessClient.getRuntimeSettings().platform);
+    formData.set('email', formValues.email);
+    formData.set('company', formValues.company);
+    formData.set('use-case', formValues.feedback);
+    formData.set('newsletter-opt-in', formValues.newsletterEnabled ? 'y' : 'n');
+    formData.set('sales-opt-in', formValues.salesContactEnabled ? 'y' : 'n');
+
+    const headers = new Headers();
+    headers.set('content-type', 'multipart/form-data');
+
+    const response = await fetch(
+      'https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod',
```

Comment:
How do you plan to provide different URLs here?

Reply:
I added 
```typescript  
const feedbackUrl = ctx.mainProcessClient.getRuntimeSettings().dev
    ? 'https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod'
    : 'https://usage.teleport.dev';
```
what do you think?

Reply:
Cool for now, but in the future let's see if we can utilize another mechanism for this. Usually I'd do it through some kind of env variables inserted during compilation, in general we need some kind of config that will be different for dev vs prod.